src/licenses.ts

```diff
@@ -24,11 +25,40 @@ export async function getInvalidLicenseChanges(
   licenses: {
     allow?: string[]
     deny?: string[]
+    licenseExclusions?: string[]
   }
 ): Promise<InvalidLicenseChanges> {
   const {allow, deny} = licenses
+  const licenseExclusions = licenses.licenseExclusions?.map(
+    (pkgUrl: string) => {
+      return PackageURL.fromString(pkgUrl)
+    }
+  )
 
   const groupedChanges = await groupChanges(changes)
+
+  // filter out changes that are part of exclusions list - config.allow_dependencies_licenses
+  groupedChanges.licensed = groupedChanges.licensed.filter(change => {
+    const changeAsPackageURL = new PackageURL(
+      change.ecosystem,
+      undefined,
+      change.name,
+      change.version,
+      undefined,
+      undefined
+    )
```

Comment:
Can we get rid of these hardcoded `undefined` values?

Reply:
Switched to creating this from the `package_url` string in [7b5fa84](https://github.com/actions/dependency-review-action/pull/423/commits/7b5fa84cfcb67839b0649ae5c652d657a9912cf6).